src/harness/testcases/testdata/ppa_record_2.json

```diff
@@ -20,18 +20,18 @@
 					-88.82560,
 					30.72540 
 				  ],
-				  [
-					  -88.66220,
-					30.71010
-				  ],
-				  [
-						-88.68280,
-						30.56470
-				  ],
 				  [
 						-88.8407,
 						30.5848
 				  ],
+          [
```

Comment:
Please make the indentation consistent with the other lat/long points.

Reply:
I updated the indentations of the zone section, is it ok like this?